jobserver/forms.py

```diff
@@ -9,14 +9,18 @@ class JobCreateForm(forms.ModelForm):
     class Meta:
 
         fields = [
-            "workspace",
             "force_run",
             "force_run_dependencies",
             "action_id",
             "backend",
             "callback_url",
         ]
         model = Job
+        widgets = {
+            "action_id": forms.TextInput(),
+            "backend": forms.TextInput(),
```

Comment:
Can we make this a choice field? (`tpp`, `emis`, `all` -- it can default to `all` and things will work as-is)

Reply:
Fixed.  `JobRequest` now has a choices driven text field with the model introducing the `all` key (and cleaning it such that `backends` is always a list), so the view can iterate over `backends` building up the required number of `JobRequests`.